src/mixins/gameMixin.js

```diff
@@ -130,11 +128,11 @@ export default {
       }
     },
     ytPaused() {
-      console.log("pasued");
+      Logger.log("pasued");
       if (this.started) this.pauseGame();
     },
     ytError() {
-      console.error("youtube error");
+      Logger.error("youtube error");
```

Comment:
Perhaps you'd want a bit more info here. Why did this error occur? YouTube is down? Auth failed? Rate limit reached? etc.

Reply:
Make sense, added

Reply:
I have found out that the function doesn't provide error detail so I have reverted the change...

Reply:
Well you could still try to serialize the error somehow probably?

Reply:
But... how?

Reply:
Ah see this method `ytError` doesn't even take any parameters, so you can't even try to serialize something you don't have (aka the error cause and context) lol

Reply:
So you mean like throw a custom error?